oap-server/server-starter/pom.xml

```diff
@@ -171,6 +171,11 @@
             <artifactId>skywalking-telegraf-receiver-plugin</artifactId>
             <version>${project.version}</version>
         </dependency>
+        <dependency>
+            <groupId>org.apache.skywalking</groupId>
+            <artifactId>aws-firehose-receiver</artifactId>
```

Comment:
You should update dependencies and licenses, as new jars are added.

Reply:
Dependencies and licenses?  `aws-firehose-receiver` is a new module of SkyWalking, and it only depends on `otel-receiver-plugin`. Please tell me where I should update.

Reply:
I saw dependencies check fails, if you don't add new dependencies, then we need to check why CI fails.